### ISSUE TITLE
Quick Tags: Close popover after adding tags

### ISSUE THREAD

**Issue by AprilSylph** [AprilSylph/XKit-Rewritten#1183]
Like with Quick Reblog...
- when the POST request is pending: disable all inputs, show some sort of progress notification covering the popover
- when the POST request is complete: hide the popover, re-enable all inputs

### Discussed in https://github.com/AprilSylph/XKit-Rewritten/discussions/1182

<div type='discussions-op-text'>

<sup>Originally posted by **TemptTheFates** August 10, 2023</sup>
I'm not sure if this is a bug or even an issue at all but when using quick tags with the xkit rewritten extension, it does not close automatically when clicking elsewhere or by using the 'esc' button (or even when typing something into a post, thus getting in the way of what is being written). It can only be closed by clicking on the icon for it again. The same applies for adding tags via the quick tag extension on dash (and elsewhere where you can add tags via the quick tag option).

Another instance of this being an issue is when the quick tag menu popup is opened and the post is closed (without closing the quick tag menu), if you make another post, the popup menu for it will still be there (unless you hard refresh the page).</div>

**Comment by marcustyphoon** [comment]
I don't think this popup should disappear on action. Quick Tags is frequently used multiple times in a row. It should disappear on a click that isn't within the popup and could potentially disappear on mouseleave, though? Does this line is thinking make sense?

**Comment by AprilSylph** [comment]
Yeah, that makes sense to me.
